ios/XMTPModule.swift

```diff
@@ -148,17 +148,18 @@ public class XMTPModule: Module {
 			}
 		}
 
-		// TODO: Support pagination
-		AsyncFunction("loadMessages") { (conversationTopic: String, conversationID: String?) -> [String] in
+		AsyncFunction("loadMessages") { (conversationTopic: String, conversationID: String?, limit: Int?, before: Double?, after: Double?) -> [String] in
 			guard let client else {
 				throw Error.noClient
 			}
 
 			guard let conversation = try await findConversation(topic: conversationTopic, conversationID: conversationID) else {
 				throw Error.conversationNotFound("no conversation found for \(conversationTopic)")
 			}
+            let beforeDate = before != nil ? Date(timeIntervalSince1970: before!) : nil
+            let afterDate = after != nil ? Date(timeIntervalSince1970: after!) : nil
 
-			return try await conversation.messages(after: Date.init(timeIntervalSince1970: 0)).map { try DecodedMessageWrapper.encode($0) }
+			return try await conversation.messages(limit: limit, before: beforeDate, after: afterDate).map { try DecodedMessageWrapper.encode($0) }
```

Comment:
I'm not sure if I did this swift thing right so going to wait till @nakajima comes back to review.